application.properties

```diff
@@ -0,0 +1,33 @@
+# Database Name
+		dbname=dashboard
+
+		# Database HostName - default is localhost
+		dbhost=localhost
+
+		# Database Port - default is 27017
+		dbport=27017
+
+		# MongoDB replicaset
+		#dbreplicaset=[false if you are not using MongoDB replicaset]
+		#dbhostport=[host1:port1,host2:port2,host3:port3]
+
+		# Database Username - default is blank
+		dbusername=dashboarduser
+
+		# Database Password - default is blank
+		dbpassword=dbpassword
```

Comment:
please refrain from storing username and password in the property files in github. Please point to the documentation mentioning that this needs to be updated.